### PR TITLE
feat- added login and signup Pages

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
1) Added Login Page
2) Added SignUp page
3) Added Navigation component  to Home page 

### Live Link :  https://deploy-preview-3--bookify-bookstore.netlify.app/

### CODE REVIEW

File: src/froentend/pages/login/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+				<div className="signup__div">
+					<form className="form">
+						<h2 className="form__name">Login</h2>
+						<div className="input__box">
+							<label>Email address </label>
+							<input id="Email" className="input" />
+							<p className="input__message">Wrong Email</p>
+						</div>
+						<div className="input__box">
+							<label>Password</label>
```
Comment (ArjunGTX) [inline src/froentend/pages/login/LoginForm.jsx]: `htmlFor` attribute can be used to bind input to corresponding label

File: src/froentend/pages/signup/SignupForm.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+function SignupForm() {
+	return (
+		<div>
+			<main className="signup__container">
+				<div className="signup__div">
+					<form action="./home.html" className="form">
+						<h2 className="form__name">Signup</h2>
+						<div className="input__box">
+							<label>Email address </label>
```
Comment (ArjunGTX) [inline src/froentend/pages/signup/SignupForm.jsx]: Same as above

**Review by ArjunGTX — APPROVED** [review]
Everything else looks good.

**Review by ishapatell1 — APPROVED** [review]

File: src/froentend/pages/login/LoginForm.jsx (hunk 1 above)
Comment (prathmeshjagtap) [reply src/froentend/pages/login/LoginForm.jsx]: yes added htmlFor to labels

File: src/froentend/pages/signup/SignupForm.jsx (hunk 2 above)
Comment (prathmeshjagtap) [reply src/froentend/pages/signup/SignupForm.jsx]: done

**Pull request merged**